File: src/targets/java/unirest/fixtures/multipart-data.java
HttpResponse<String> response = Unirest.post("http://mockbin.com/har")
  .header("content-type", "multipart/form-data; boundary=---011000010111000001101001")
  .body("-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"foo\"; filename=\"hello.txt\"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001\r\nContent-Disposition: form-data; name=\"bar\"\r\n\r\nBonjour le monde\r\n-----011000010111000001101001--\r\n")
  .asString();

File: src/targets/node/unirest/fixtures/multipart-data.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'content-type': 'multipart/form-data; boundary=---011000010111000001101001'
});

req.multipart([
  {
    body: 'Hello World',
    'content-type': 'text/plain'
  },
  {
    body: 'Bonjour le monde'
  }
]);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});